spec/pegmatite_spec.cr

```diff
@@ -19,50 +19,50 @@ describe Pegmatite do
     tokens = Pegmatite.tokenize(Fixtures::JSONGrammar, source)
     tokens.should eq [
       {:object, 0, 182},
-        {:pair, 4, 20},
-          {:string, 5, 10}, # "hello"
-          {:string, 14, 19}, # "world"
-        {:pair, 24, 180},
-          {:string, 25, 29}, # "from"
-          {:object, 32, 180},
-        {:pair, 38, 57},
-          {:string, 39, 43}, # "name"
-          {:string, 47, 56}, # "Pegmatite"
-        {:pair, 63, 83},
-          {:string, 64, 71}, # "version"
-          {:array, 74, 83},
-            {:number, 75, 76}, # 0
-            {:number, 78, 79}, # 1
-            {:number, 81, 82}, # 0
-        {:pair, 89, 102},
-          {:string, 90, 95}, # "nifty"
-          {:true, 98, 102}, # true
-        {:pair, 108, 132},
-          {:string, 109, 124}, # "overcomplicated"
-          {:false, 127, 132}, # false
-        {:pair, 138, 156},
-          {:string, 139, 149}, # "worse-than"
-          {:null, 152, 156}, # null
-        {:pair, 162, 176},
-          {:string, 163, 171}, # "problems"
-          {:array, 174, 176}, # []
+      {:pair, 4, 20},
+      {:string, 5, 10},  # "hello"
+      {:string, 14, 19}, # "world"
+      {:pair, 24, 180},
+      {:string, 25, 29}, # "from"
+      {:object, 32, 180},
+      {:pair, 38, 57},
+      {:string, 39, 43}, # "name"
+      {:string, 47, 56}, # "Pegmatite"
+      {:pair, 63, 83},
+      {:string, 64, 71}, # "version"
+      {:array, 74, 83},
+      {:number, 75, 76}, # 0
+      {:number, 78, 79}, # 1
+      {:number, 81, 82}, # 0
+      {:pair, 89, 102},
+      {:string, 90, 95}, # "nifty"
+      {:true, 98, 102},  # true
+      {:pair, 108, 132},
+      {:string, 109, 124}, # "overcomplicated"
+      {:false, 127, 132},  # false
+      {:pair, 138, 156},
+      {:string, 139, 149}, # "worse-than"
+      {:null, 152, 156},   # null
+      {:pair, 162, 176},
+      {:string, 163, 171}, # "problems"
+      {:array, 174, 176},  # []
```

Comment:
The loss of these layers of indentation is unfortunate (it makes it much harder to see what's going on), but I'm not sure it's important enough for me to actually oppose the change.